tests/ieee802154_security/Makefile

```diff
@@ -0,0 +1,3 @@
+USEMODULE += ieee802154_security
+
```

Comment:
```suggestion
USEMODULE += gnrc_netdev_default
```

Reply:
```
USEMODULE += ieee802154
USEMODULE += ieee802154_security
USEMODULE += gnrc_netif_ieee802154
```
I noticed, it was not actually compile. With this, relevant parts are compiled.
Should `ieee802154_security` pull in `ieee802154` ?

Reply:
I think it's fine the way it is.
`ieee802154_security` is more like an option on top of `ieee802154` and `ieee802154` is usually not selected by the application.